IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h

```diff
@@ -177,7 +177,7 @@
  @param image Image icon to use as right button.
  @param target Target object for selector.
  @param action Right button action name. Usually 'doneAction:(IQBarButtonItem*)item'.
- @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
+ @param showPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
```

Comment:
Should be `shouldShowPlaceholder` because method name has `shouldShowPlaceholder` regardless of variable name.